func_map.go

```diff
@@ -244,7 +244,7 @@ func (context *Context) URLFor(value interface{}, resources ...*Resource) string
 
 			for _, primaryField := range res.PrimaryFields {
 				if field, ok := scope.FieldByName(primaryField.Name); ok {
-					primaryFields = append(primaryFields, fmt.Sprint(field.Field.Interface())) // TODO improve me
+					primaryFields = append(primaryFields, url.QueryEscape(fmt.Sprint(field.Field.Interface())))
```

Comment:
hi @raven-chen 

I found that the value of `primaryFields ` is used for the `urlpath` variable later, should we use the `url.PathEscape`  method?
For example, the space should be encoded as `%20` instead of  `+`

<img width="896" alt="Screen Shot 2020-09-25 at 14 30 19" src="https://user-images.githubusercontent.com/7029256/94234403-31645100-ff3c-11ea-8ab8-e994136b80cf.png">

Reply:
You're right! sorry, I missed your comment. made a fix https://github.com/qor/admin/pull/237 here.